tests/test_610-awkward-form.py

```diff
@@ -0,0 +1,590 @@
+import json
```

Comment:
First line should be point to the LICENSE.

These tests are probably the same as the last 10 or so in the new test file I made, so those will probably automatically work. (No reason to avoid the duplication, either. You can test the same thing here and there.)